src/components/home/welcome/LeftPanel.tsx

```diff
@@ -0,0 +1,50 @@
+import smiliesIcon from 'assets/images/smilies.png'
+import ovalIcon from 'assets/images/oval.png'
+
+import 'components/home/welcome/LeftPanel.css'
+
+const leftPanel = () => {
+  return(
+    <div className='home--base-section'>
+      <div className="home--principal-icon-row">
+        <img src={smiliesIcon} alt="Two faces smiling" className="home--principal-icon" />
+      </div>
+
+      <div className="home--title-row">
+        <p className="home--title-text">Welcome to <b>TARGET</b> </p>
+      </div>
+
+      <div className="home--title-row">
+        <p className="home--subtitle-text"><b>Find people near you & Connect</b></p>
+      </div>
+      
+      <div className="home--parragraphs-section">
+        <p className="home--parragraph-base">
+          <div className="home--parragraph-icon">
+            <img src={ovalIcon} alt="Oval icon" />
+          </div>
+          <div>
+            Create a  target  by clicking wherever on the map, specify  the ratio and and a topic: Travel, Dating, Music, etc.
+          </div>
```

Comment:
Whenever you want to add texts, you should use specific tags for it: like p, span etc. It's more semantic, WDYT?
```suggestion
          <p>
            Create a  target  by clicking wherever on the map, specify  the ratio and and a topic: Travel, Dating, Music, etc.
          </p>
```

Reply:
Great, didn't know that was required 😄